rustworkx-core/src/connectivity/biconnected.rs

```diff
@@ -68,17 +72,20 @@ fn is_root(parent: &[usize], u: usize) -> bool {
 /// ]);
 ///
 /// let mut bicomp = HashMap::new();
-/// let a_points = articulation_points(&graph, Some(&mut bicomp));
+/// let mut bridges = HashSet::new();
+/// let a_points = articulation_points(&graph, Some(&mut bicomp), Some(&mut bridges));
 ///
 /// assert_eq!(a_points, HashSet::from_iter([nx(1)]));
 /// assert_eq!(bicomp, HashMap::from_iter([
 ///     ((nx(0), nx(2)), 1), ((nx(2), nx(1)), 1), ((nx(1), nx(0)), 1),
 ///     ((nx(1), nx(3)), 0)
 /// ]));
+/// assert_eq!(bridges, HashSet::from_iter([(nx(1), nx(3))]));
 /// ```
 pub fn articulation_points<G>(
     graph: G,
     components: Option<&mut HashMap<Edge<G>, usize>>,
+    bridges: Option<&mut HashSet<Edge<G>>>,
```

Comment:
Can you keep the signature of `articulation_point` the same and create an internal function `_articulation_point` that is called by both `articulation_point` and `bridges`?